specification/theory_operations.adoc

```diff
@@ -314,6 +314,7 @@ The TSM is responsible for:
 The host VMM is responsible for:
 1. Generating and managing system wide PCIe stream IDs.
 2. Setting the device PCIe device IDE PCI Extended Capability.
+3. Programing the switch between the device and PCIe root port, such as IDE Control Register.
```

Comment:
Do you mean the PCIe switch between the device and the RP?

Reply:
Yes, I updated it to "PCIe switch"